cmd/goodman/main.go

```diff
@@ -51,7 +51,7 @@ func main() {
 				}
 			}()
 			for retries := 5; retries > 0; retries-- {
-				runner, err := goodman.NewRunner("Hooks", hookServerInitalPort)
+				runner, err := goodman.NewRunner("HooksRunner", hookServerInitalPort)
```

Comment:
Yep, that was the cause for the timeouts.

I've pushed the change to retrieve the struct name via reflection.